webpack.common.js

```diff
@@ -1,14 +1,9 @@
-var webpack = require('webpack')
-var path = require('path')
-var HtmlWebpackPlugin = require('html-webpack-plugin')
-var CopyWebpackPlugin = require('copy-webpack-plugin')
+const webpack = require('webpack')
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
 const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 
-// Fix possible nofile-issues
-var fs = require('fs')
-var gracefulFs = require('graceful-fs')
-gracefulFs.gracefulify(fs)
```

Comment:
I would like to keep this.

![image](https://user-images.githubusercontent.com/8719867/47156196-dacd5180-d2e6-11e8-9438-8eaa6e54626c.png)

Reply:
Ahh, that's what this is for ^^

Let's keep removing this out of this PR then. Do we have an open Issue to get to the bottom of the error?

Reply:
I don't think so.

https://github.com/systemd/systemd/pull/10244 might solves it soon, without touching any (Node)JS code :smile: 

Apart from that, I guess it's simply related to the size of our codebase :/